src/main/scala/au/com/agiledigital/toolform/command/generate/docker/GenerateDockerComposeV3Command.scala

```diff
@@ -69,12 +68,11 @@ object GenerateDockerComposeV3Command extends YamlWriter {
         _ <- write("version: '3'")
         _ <- write("services:")
         _ <- indented {
-          for {
-            _ <- writeEdges(project.id, subEdgeDefsFromProject(project).toList)
-            _ <- writeComponents(project.id, project.sortedComponents.values.toList)
-            _ <- writeResources(project.sortedResources.values.toList)
-          } yield ()
-        }
+              for {
```

Comment:
A bit of sugar to get rid of this would go a long way. No change required for this PR, but something to ponder on.